common.html

```diff
@@ -0,0 +1,29 @@
+<script>
+function ShadeyCssCommon() {
+  'use strict';
+
+  var matcher = {
+    whitespace: /\s/,
```

Comment:
It might be safe to make sure the classifier patterns don't overlap, so that you don't get changes in behavior if you evaluate them in a different order. I would move grouping tokens together (merge into and rename blockBoundary?) and remove propertyDelimiter.